test_runner/src/main/kotlin/ftl/args/AndroidArgs.kt

```diff
@@ -55,19 +55,22 @@ class AndroidArgs(
     private val androidGcloud = androidGcloudYml.gcloud
     var appApk = (cli?.app ?: androidGcloud.app ?: throw FlankFatalError("app is not set")).processFilePath("from app")
     var testApk = (cli?.test ?: androidGcloud.test)?.processFilePath("from test")
-    val additionalApks = (cli?.additionalApks ?: androidGcloud.additionalApks).map { it.processFilePath("from additional-apks") }
+    val additionalApks =
```

Comment:
Pls let's try to avoid autoformat changes

Reply:
Right! I set auto line breaking to 200 chars but this changes was made before. It shouldn't happen any more.